package.json

```diff
@@ -1,30 +1,26 @@
 {
-  "name": "nvquicktheme",
-  "version": "2.0.0",
-  "description": "Barebones Bootstrap 4 DNN Theme",
-  "main": "gulpfile.js",
-  "author": "nvisionative",
-  "license": "GPL-3.0",
-  "private": true,
+  "name": "40FINGERS-DNN-Theme",
+  "version": "1.0.0",
+  "description": "40FINGERS DNN Theme",
+  "main": "default.aspx",
   "dependencies": {
-    "bootstrap": "^4.6.2",
-    "font-awesome": "^4.7.0",
-    "normalize.css": "^8.0.1"
+    "@fortawesome/fontawesome-free": "^5.1.0-9",
+    "bootstrap": "^5.3.2",
+    "gulp-rename": "^2.0.0"
```

Comment:
Not a huge issue, but this should be in `devDependencies` instead of `dependencies` since it is not distributed.

Reply:
Apologies if this one was not clear @Timo-Breumelhof - I am referring to `gulp-rename`.  That package should be in `devDependencies` instead of `dependencies`.  Thanks!

Reply:
I also noticed 40FINGERS in there, changed that to DNNCommunity